src/psm/include/psm/pdnsim.h

```diff
@@ -69,6 +72,7 @@ class PDNSim
   void set_pdnsim_net_voltage(std::string net, float voltage);
   int  analyze_power_grid();
   void write_pg_spice();
+  void getIRDropMap(std::map<odb::dbTechLayer*, std::map<odb::Point ,double>>& ir_drop);
```

Comment:
would it be possible provide an accessor function for the minimum length? (in DBU?)
maybe something like `int getMinimumResolution()` that will help keep the IR heatmap from generating cells smaller than the 2.1um limit. I could hardcode that in to the IR drop map, but it might be better to keep the value in PSM so I doesn't have to change in more than 1 place if it changes in the future.

Reply:
@arlpetergadfort  I guess when you say minimum length you mean what the smallest distance between two nodes (points) is across all layers? Is this correct? I am not sure what the 2.1um limit is.

Reply:
That's correct, the minimum distance. The 2.1um might have been wrong.